src/nodes/function_expression.js

```diff
@@ -34,6 +34,10 @@ export function format(node, context, recur, {noFunctionKeyword} = {}) {
         context.write('function');
     }
 
+    if (node.generator) {
+        context.write('* ');
```

Comment:
interesting... i would have these styles for generators

```js
funciton* a() {}

function *a() {}

let a = function*() {}
```

but i'm not sure which one would be more correct :)